Accordion.js

```diff
@@ -71,20 +74,23 @@ export default class Accordion extends Component {
       }
     });
 
+    const Touchable = this.props.touchableComponent;
+
     return (
       <View {...viewProps}>
         {this.props.sections.map((section, key) =>
           <View key={key}>
-            <TouchableHighlight
+            <Touchable
               onPress={() => this._toggleSection(key)}
               underlayColor={this.props.underlayColor}
+              {...this.props.touchableProps}
```

Comment:
I believe this needs to have a default of empty object otherwise it will crash for `undefined` values. Did you try this code runtime?